hw/dv/sv/uart_agent/uart_agent.core

```diff
@@ -18,6 +18,7 @@ filesets:
       - uart_driver.sv: {is_include_file: true}
       - uart_monitor.sv: {is_include_file: true}
       - uart_sequencer.sv: {is_include_file: true}
+      - uart_logger.sv: {is_include_file: true}
```

Comment:
@weicaiyang 
Note that I moved the logger to uart_agent. Advantage is universal availability.

Reply:
sounds good